chrome/updater/configurator.cc

```diff
@@ -107,6 +109,12 @@ int Configurator::UpdateDelay() const {
 }
 
 std::vector<GURL> Configurator::UpdateUrl() const {
+// TODO(b/325626249): Remove the ALLOW_EVERGREEN_SIDELOADING check once we're fully launched.
+#if !defined(COBALT_BUILD_TYPE_GOLD) && ALLOW_EVERGREEN_SIDELOADING
```

Comment:
maybe im missing this, is this macro set to false somewhere ? 

#define ALLOW_EVERGREEN_SIDELOADING 0

Reply:
Apart from this , LGTM 

My understanding is apart from this flag this is asimple cherry-pick of the already reviewed change in trunk

Reply:
Unresolved comment